app/lib/models.dart

```diff
@@ -86,6 +93,19 @@ class Package extends db.ExpandoModel {
       latestDevVersionKey = pv.key;
     }
   }
+
+  /// Updates the search-related fields (eg. [scope]) to match the latest stable
+  /// [PackageVersion].
+  ///
+  /// Returns whether any field has been updated.
+  bool updateSearchFields(PackageVersion pv) {
```

Comment:
Discussed parts of this offline. Let's wait until the search/indexing solution is decided, because that would make the question of naming irrelevant here.